Gatekeeper.Client/wwwroot/img/favicons/site.webmanifest

```diff
@@ -1,19 +1,19 @@
 {
-    "name": "Gatekeeper",
-    "short_name": "Gatekeeper",
+    "name": "",
+    "short_name": "",
     "icons": [
         {
-            "src": "/img/favicons/android-chrome-192x192.png",
+            "src": "/android-chrome-192x192.png",
             "sizes": "192x192",
             "type": "image/png"
         },
         {
-            "src": "/img/favicons/android-chrome-512x512.png",
+            "src": "/android-chrome-512x512.png",
             "sizes": "512x512",
             "type": "image/png"
         }
     ],
-    "theme_color": "#034f84",
-    "background_color": "#034f84",
+    "theme_color": "#ffffff",
```

Comment:
Intentional? :)

Reply:
@LukasReschke not sure, can update later if it's an issue :P